pkg/fixedpoint/convert.go

```diff
@@ -430,6 +430,10 @@ func (a Value) MulExp(exp int) Value {
 	return Value(int64(float64(a) * math.Pow(10, float64(exp))))
 }
 
+func (a Value) MulPow(v Value, exp Value) Value {
+	return Value(int64(float64(a) * math.Pow(v.Float64(), exp.Float64())))
```

Comment:
You need also implement the same method for dnum fixedpoint.  
Otherwise strategies compiled using dnum tag won't be able to pass the compilation